### PR TITLE
Fix/styles vision 2

### PR DESCRIPTION
**Pull request by simaob**

### CODE REVIEW

**Comment by simaob** [comment]
@j8seangel review this one please =D

File: app/components/pages/HomePage.js [hunk 1]
```diff
@@ -5,33 +5,33 @@ function HomePage() {
...
               <use xlinkHref="#illustration-3"></use>
             </svg>
-            <h3 className="header -intro">You can also search for a species and see its critical sites</h3>
+            <h3 className="header -intro">Search for a species<br />and see its critical sites</h3>
           </li>
         </ul>
         <div className="column small-12 medium-6 medium-offset-3">
           <p className="text -intro">The tool makes it easy to obtain information on the sites critical for waterbird species by accessing several independent databases and analysing information at the biogeographical population level, so providing a comprehensive basis for management and decision making. It is designed to help a range of different users from site managers to national authorities and international organisations. The CSN Tool, one of the major achievements of the Wings Over Wetlands (WOW) project, is also an important example of the added value of cooperation between likeminded conservation organisations, international conventions and agreements, governments, UN agencies and other donors.</p>
-
+          <br />
```
Comment (j8seangel) [inline app/components/pages/HomePage.js]: Do we really need this ? What do you think about add margin to the <p> tags ?

File: app/styles/components/home/c-banner.pcss [hunk 2]
```diff
@@ -16,8 +15,7 @@
     align-items: center;
     justify-content: center;
     flex-direction: column;
-    z-index: 1;
-    background-color: rgba($black, 0.5);
+    background-color: rgba(#000, 0.5);
```
Comment (j8seangel) [inline app/styles/components/home/c-banner.pcss]: We prefer to use the constants variables, could you please change it ?

**Comment by simaob** [comment]
See: https://github.com/Vizzuality/csn-tool/pull/58

**Pull request closed without merging**